bpf/lib/common.h

```diff
@@ -221,9 +221,8 @@ struct endpoint_key {
 	union {
 		struct {
 			__u32		ip4;
-			__u32		pad1;
-			__u32		pad2;
-			__u32		pad3;
+			__u32		vni;
```

Comment:
I think the intent here is to change the value of the map, but today we're reusing the same structure for both key and value. We should probably split this into two separate structures and only change the value so that it's very clear that the MAC is not used as a lookup key and we're not expecting the map to be used that way.

Reply:
My initial intent is to make the change as minimal as possible and not to affect the rest code, would separating the structures bring more intrusive changes?  I had some problem with bpf/bpf_alignchecker.c when I made more changes in the endpoint_key structure, do you really want me to try that and see how it goes ? :)